device-types/Lenovo/system-x3650-M5.yaml

```diff
@@ -0,0 +1,60 @@
+---
+manufacturer: Lenovo
+model: System x3650 M5
+slug: lenovo-system-x3650-m5
+part_number: 8871
+u_height: 2
+is_full_depth: true
+comments: '[Lenovo System x3650 M5 product guide](https://lenovopress.com/tips1194.pdf)'
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+  - name: PSU 2
+    type: iec-60320-c14
+rear-ports:
```

Comment:
Rear ports are designed for patch panels not devices, standard USB ports on servers are not something that we are currently mapping in this repository